conans/client/tools/files.py

```diff
@@ -363,3 +363,14 @@ def unix2dos(filepath):
 
 def dos2unix(filepath):
     _replace_with_separator(filepath, "\n")
+
+
+def dot_clean(folder):
```

Comment:
I would name it `dot_underscore_clean`

Reply:
There is a built-in utility in Mac called `dot_clean`. I wanted to keep the same naming, and then you will ask me: "Why don't we use the command line?", and I will say 😒 ...

```
⇒  dot_clean -h    
usage: dot_clean  [-fhmnpsv] [--keep=[mostrecent|dotbar|native]] [directory ...]

 f : flat.	Disables recursive conversion.
 h : help.	Print this message.
 m : Always delete apple double files.
 n : Delete apple double file if there is no matching native file.
 p : Preserve apple double file
 s : Follow symbolic links.
 v : Verbose

 keep : Specify whether to keep the mostrecent, dotbar, or native.

```

Reply:
Then call it "mac_dot_clean" :P

Reply:
Shall I move it to `conans.client.tools.apple` and call it `apple_dot_clean`?